README.md

```diff
@@ -37,17 +37,33 @@ Location of the YAML config file to load.
 
 Log verbosity level. Defaults to `info`. Use `debug` to get more details.
 
+### WAN Interface Statistics
+
+- Config: `interfaces`
+- Args: `--with-interfaces`
+- Env: `UNMS_EXPORTER_INTERFACES`
+
+Enable exporting WAN interface metrics. Defaults to `true`.
```

Comment:
the name of the flag suggests that it's about _all_ interfaces, but the descriptions just states _WAN_. which is correct?

Reply:
True. I've derived the name from the `defaultWithInterfaces` variable, but I'm not keen on that name either.

What do you think of `--with-wan-statistics`/`UNMS_EXPORTER_WAN_STATISTICS`?

Reply:
I'm going with `--enable-wan-metrics`.